src/Microsoft.AspNetCore.Mvc.ViewFeatures/Rendering/TagBuilder.cs

```diff
@@ -343,6 +329,110 @@ private string DebuggerToString()
             }
         }
 
+        /// <summary>
+        /// Returns the <see cref="IHtmlContent"/> body.
+        /// </summary>
+        /// <returns>The inner <see cref="IHtmlContent"/>.</returns>
+        public IHtmlContent RenderBody()
+        {
+            return _innerHtml;
```

Comment:
Should it return an empty `IHtmlContent` if it's null? So that we can always call it without having to check id the result is `null`.